lib/provider/vpc_data_types.go

```diff
@@ -50,6 +54,20 @@ type VPC struct {
 	Name string `json:"name,omitempty"`
 }
 
+// SecurityGroup ...
+type SecurityGroup struct {
+	ID   string `json:"id"`
+	CRN  string `json:"crn,omitempty"`
+	Href string `json:"href,omitempty"`
+}
+
+// PrimaryIP ...
+type PrimaryIP struct {
```

Comment:
HREF can be added

Reply:
Can you add href for all resource struct